.github/workflows/build.yml

```diff
@@ -5,14 +5,24 @@ on: [push, pull_request]
 jobs:
   build-macos:
     runs-on: macOS-latest
+    env:
+      CCACHE_COMPRESS: 1
+      CCACHE_TEMPDIR: /tmp/.ccache-temp
     steps:
     - uses: actions/checkout@v1
       with:
         submodules: recursive
+    - uses: actions/cache@v2
+      with:
+        path: ~/.ccache
+        key: ccache-macos-build-${{ github.sha }}
```

Comment:
Was the ${{ github.sha }} suffix missing previously?

Reply:
I don't think so.